### ISSUE TITLE
AppStream errors

### ISSUE THREAD

**Issue by darkdragon-001** [flathub/com.github.PintaProject.Pinta#39]
FlatHub builder shows the following AppStream meta info errors and blocks updates:
```
{
    "errors": [
        "appstream-failed-validation"
    ],
    "appstream": [
        "E: com.github.PintaProject.Pinta:99: description-para-markup-invalid ul",
        "E: com.github.PintaProject.Pinta:92: description-para-markup-invalid ul",
        "E: com.github.PintaProject.Pinta:97: description-para-markup-invalid ul",
        "E: com.github.PintaProject.Pinta:94: description-para-markup-invalid ul"
    ],
    "message": "Please consult the documentation at https://docs.flathub.org/docs/for-app-authors/linter"
}
```

@cameronwhite could you please take a look?

**Comment by cameronwhite** [comment]
The line numbers didn't make much sense to me, but running `appstreamcli validate xdg/pinta.appdata.xml.in` locally I got 
```
W: pinta.desktop:3: cid-desktopapp-is-not-rdns pinta.desktop
E: pinta.desktop:9: description-markup-invalid _p
I: pinta.desktop:19: nonstandard-gnome-extension kudos
I: pinta.desktop:29: developer-name-tag-deprecated
E: pinta.desktop:69: description-para-markup-invalid ul
E: pinta.desktop:74: description-para-markup-invalid ul
E: pinta.desktop:79: description-para-markup-invalid ul
E: pinta.desktop:84: description-para-markup-invalid ul
E: pinta.desktop:~: app-description-required
I: pinta.desktop:~: developer-info-missing

✘ Validation failed: errors: 6, warnings: 1, infos: 3
```

The error message isn't very descriptive, but I think it doesn't like the nested lists in the release notes (nested `ul` elements)

**Comment by darkdragon-001** [comment]
@cameronwhite thanks for checking. I would expect some of the errors arising from the `.in` template.

Your error is [`description-para-markup-invalid`](https://www.freedesktop.org/software/appstream/docs/chap-Validation.html#asv-description-para-markup-invalid) and here is the specification for [`<description/>`](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-description) stating that "Nested lists are not supported".

**Comment by cameronwhite** [comment]
Is it easy to apply a patch here to fix it, or is a new release required?

**Comment by darkdragon-001** [comment]
You can also create a patch here. Just add a patch file (here `file.patch`) and reference it in an additional element in your sources in the manifest:
```yaml
name: Pinta
sources:
  - type: patch
    path: file.patch
```

See [here](https://github.com/flathub/org.gnome.Calls/blob/7e190d6d1d2b2f3ffaa043cc211ad59730c5e57a/org.gnome.Calls.yaml#L263-L264) for an example.

**Comment by cameronwhite** [comment]
Should be fixed by #40

**Comment by darkdragon-001** [comment]
Awesome, thanks!
